src/mono/wasm/debugger/DebuggerTestSuite/DebuggerTestFirefox.cs

```diff
@@ -37,7 +37,7 @@ public override async Task InitializeAsync()
             };
 
         await Ready();
-        await insp.OpenSessionAsync(fn, TestTimeout);
+        await insp.OpenSessionAsync(fn, TestTimeout*4);
```

Comment:
Can we save this info as a fixed timeout for Firefox? I mean: 

```
public class DebuggerTestBase
{
    private const int DefaultTestTimeoutMs = 1 * 60 * 1000;
    protected TimeSpan TestTimeout => TimeSpan.FromMilliseconds(DefaultTestTimeoutMs);
}

public class DebuggerTestFirefox : DebuggerTestBase
{
    private new TimeSpan TestTimeout => base.TestTimeout * 4;
}
```

```suggestion
        await insp.OpenSessionAsync(fn, TestTimeout);
```